Dockerfile

```
@@ -1,46 +1,50 @@
-FROM golang:1.19.4-alpine AS operator-build
-
-RUN apk update --no-cache && \
-    apk add --no-cache gcc musl-dev btrfs-progs-dev lvm2-dev device-mapper-static gpgme-dev git && \
-    rm -rf /var/cache/apk/*
+FROM golang:1.19.4 AS operator-build
 
 ARG GO_LINKER_ARGS
+
 COPY . /app
 WORKDIR /app
 
-# move previously cached go modules to gopath
-RUN if [ -d ./mod ]; then mkdir -p ${GOPATH}/pkg && [ -d mod ] && mv ./mod ${GOPATH}/pkg; fi;
+RUN apt-get update && \
+    apt-get install -y libbtrfs-dev libdevmapper-dev
 
-RUN CGO_ENABLED=1 CGO_CFLAGS="-O2 -Wno-return-local-addr" go build -ldflags="${GO_LINKER_ARGS}" -o ./build/_output/bin/dynatrace-operator ./src/cmd/
+RUN CGO_ENABLED=1 CGO_CFLAGS="-O2 -Wno-return-local-addr" \
+    go build -tags "containers_image_openpgp" -ldflags="${GO_LINKER_ARGS}" \
+    -o ./build/_output/bin/dynatrace-operator ./src/cmd/
 
 FROM registry.access.redhat.com/ubi9-minimal:9.1.0 as dependency-src
 
-RUN  microdnf install util-linux tar --nodocs -y && microdnf clean all
+RUN microdnf install -y util-linux tar --nodocs
 
 FROM registry.access.redhat.com/ubi9-micro:9.1.0
 
-# operator dependencies
-COPY --from=operator-build /etc/ssl/cert.pem /etc/ssl/cert.pem
+# operator binary
 COPY --from=operator-build /app/build/_output/bin /usr/local/bin
 
-COPY --from=operator-build /lib/libc.musl-*.so.* /lib/
-COPY --from=operator-build /lib/ld-musl-*.so.* /lib/
+# cgo dependencies
+COPY --from=operator-build /usr/lib/*/libdevmapper.so.* /usr/lib/
+COPY --from=operator-build /lib/*/libdevmapper.so.* /lib/
```

```suggestion
COPY --from=operator-build /lib/*/libdevmapper.so* /lib/
```
`*.so` files don't match `so.*` pattern